sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -729,6 +743,25 @@ def schedules(self) -> ScheduleOperations:
         """
         return self._schedules
 
+    @property
+    @experimental
+    def featuresets(self) -> FeaturesetOperations:
+        """A collection of featureset related operations.
+
+        :return: Featureset operations
+        :rtype: FeaturesetOperations
+        """
+        return self._featuresets
+
+    @property
+    def featurestore_entities(self) -> FeaturestoreEntityOperations:
```

Comment:
property name is changed to featurestore_entities, in spec it is featurestoreEntities (It was giving lint warning for name). In sdk snake casing is preffered

Reply:
cc: @qjxu

Reply:
It looks good to me. @singankit, Is above naming convention matches our guidance?